packages/react-environment/src/types/mixer-config.interface.ts

```diff
@@ -0,0 +1,10 @@
+export interface TornMixerEntry {
+  size: number;
+  address: string;
+  symbol: string;
```

Comment:
@nepoche  do you think it worth having the `WebbCurrencyId` instead of the symbol string?